packages/clay-dropdown/src/ClayDropdownBase.soy

```diff
@@ -135,7 +134,7 @@
 		"
 	{/let}
 
-	<ul {$listAttributes}>
+	<div {$listAttributes}>
```

Comment:
why are we moving away from semantic tags?

Reply:
`<ul>` elements on no account must incorporate elements other than `<li>`, `<script>`, `<template>`. (reference - https://dequeuniversity.com/rules/axe/1.1/list)

A solution would be to append all content wrapped by `<li>` elements, but due the code there seems to wrap everything in `<div>` I thought I'd follow that approach and remove `<ul>` completely. This change could likely imply unexpected design consequences though.

Reply:
If this is a list of items, it needs to be a list. If an item is not an item, either it needs to be put outside the list or the list needs to be declared deeper down to enclose the real list of items.

Reply:
Agreed. With this mind, I will have to refactor the whole dropdown soy file.

Reply:
👍

Reply:
I think making all `{deltemplate ClayDropdown.item}` and variants `li` elements (and `ul` the parent) and adding `aria-hidden` and `role=presentation` to the `separator` and `header` variants should do the work.

Reply:
I'm addressing this.